SparkPost/2web2ui
# FE-1186: Fixed behavior with typeselect

### What Changed
 - Typeselect opens menu and clears filter on focus

### How To Test
 - Go to /signals/analytics (hibana)
 - On top with saved reports, select a report
 - Leave focus and focus again, should reopen menu, highlight text, and then show all results

### To Do
